### PR TITLE
Post, Sign In and Report

### PR DESCRIPTION
**Pull request by sura1542**
- Post finish 80%
- Sign In 95%
- Start Report

### CODE REVIEW

**Review by Balloii — APPROVED** [review]

File: src/views/Comment.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
     <!-- <Sidebar class="sidebar" /> -->
     <main class="comment-page">
-        <h1>Comment</h1>
+        <h1>Comment pergepr</h1>
```
Comment (aofattaporn) [inline src/views/Comment.vue]: can you convert it to the original version,  I am concerned it conflict with another branch (thun)

File: src/views/Post.vue [hunk 2]
```diff
@@ -1,14 +1,228 @@
...
+        labels: ["January", "February", "March", "April", "May", "June", "July"],
+        datasets: [{ data: [40, 20, 12] }],
+      },
+      chartOptions: {
+        responsive: true,
+      },
+    };
+  },
+};
 </script>
```
Comment (aofattaporn) [inline src/views/Post.vue]: actually, I want to convert Composition API  more than Option API, because if you use Composition you should use it anywhere.

**Review by aofattaporn — COMMENTED** [review]
actually, when you PR please describe this. 

1. what do you do 
2. if not finished, what do you have not finished please describe

**Comment by sura1542** [comment]
I fix it all finished. Please Chach again

**Review by Balloii — APPROVED** [review]

File: src/views/Report.vue [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        datasets: [{ data: [40, 20, 12] }],
+      },
+      chartOptions: {
+        responsive: true,
+      },
+    };
+  },
+};
+</script>
+
```
Comment (aofattaporn) [inline src/views/Report.vue:122]: I'm not sure this script is Composition API because it has the keyword "data" that look like an Optional API

**Review by aofattaporn — APPROVED** [review]
actually, I want to use Composition API. But at this time that sounds good for merging (LGTM). and you can resolve this next time

File: src/views/Report.vue (hunk 3 above)
Comment (aofattaporn) [reply src/views/Report.vue:122]: please to learn more if Composition API use `ref()` to collect state variable 
but Optional will use `data()` to collect state variable 

ref : https://vuejs.org/guide/introduction.html#api-styles

Comment (aofattaporn) [reply src/views/Report.vue:122]: can you merge it and resolve this problem after consulting in the team next time

**Pull request merged**